HadGEM3-GC31-LL tasmax ssp585 min value out of range

Workflow : https://argo.cildc6.org/archived-workflows/default/e46de2d6-5612-4dbf-92a6-c06060dc3b42


Log : 
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/MOHC/HadGEM3-GC31-LL/ssp585/r1i1p1f3/day/tasmax/gn/v20200114.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 47, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 32, in clear_memory_intensive_tests
    _test_temp_range(d, v)
  File "/opt/dodola/dodola/core.py", line 760, in _test_temp_range
    assert (ds[var].min() > 130) and (
AssertionError: tasmax values are invalid

```

--- a/workflows/templates/qualitycontrol-check-cmip6.yaml
+++ b/workflows/templates/qualitycontrol-check-cmip6.yaml
@@ -40,7 +40,7 @@ spec:
           - name: data
           - name: time
       script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.16.0
+        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.17.0
         command: [python]
         source: |
           # Running this as a script rather than a dodola command as workaround to 

